src/nsls2api/services/beamline_service.py

```diff
@@ -320,6 +320,12 @@ async def update_data_admins(beamline_name: str, data_admins: list[str]):
 
 
 async def directory_skeleton(name: str):
+    # Let's make sure the beamline exists
+    beamline = await beamline_by_name(name)
+    # If there's no beamline then it's not possible to generate a directory skeleton
+    if beamline is None:
+        return None
```

Comment:
Future improvement TODO: It might be worth defining a `directory` model, to be used as return type (and perhaps vaildation elsewhere?).

Reply:
You mean something like https://github.com/NSLS2/nsls2api/blob/main/src/nsls2api/models/beamlines.py#L24

Reply:
Created #119